bin/build-docker.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+/**
+ * Script to run docker build.
+ *
+ * This script is needed to enable us to embed the git SHA as a build arg
+ * in both unix shells and windows.
+ *
+ * Essentially, it doees this:
+ * docker build --build-arg commit_sha=`git rev-parse HEAD` -t wp-calypso .
+ */
+
+const { spawnSync, execSync } = require( 'child_process' );
+
+const sha = String( execSync( 'git rev-parse HEAD' ) ).trim();
+
+const args = [
+	'build',
+	'--build-arg', 'commit_sha=' + sha,
+	'-t', 'wp-calypso',
+	'.'
+];
+
+console.log( 'docker ' + args.join( ' ' ) );
+spawnSync( 'docker', args, { stdio: 'inherit' } );
```

Comment:
at this point, could this just be moved into the `scripts` section of package.json?

```bash
docker build --build-arg commit_sha=$(git rev-parse HEAD) -t wp-calypso .
```

after a little searching I found a sad Christmas 😢 so maybe this extra script is worth it

Reply:
That's what I did at first, but it's not windows-compatible.

Reply:
oh right, that's why it was sad christmas for me